### ISSUE TITLE
Add option to use pullrequest template

### ISSUE THREAD

**Issue by MiKuehn** [shayki5/azure-devops-create-pr-task#150]
It would be helpful to be able to specify a template instead of the description.

Parameter: templatePath

The path points to a template file in the Git repository.
The template path is higher ranked than the description. If the template path is defined, the description is ignored.

**Comment by shayki5** [comment]
Hi @MiKuehn, I think we can't choose a template when we creating a PR, Azure DevOps search for them automatically, no?
https://learn.microsoft.com/en-us/azure/devops/repos/git/pull-request-templates?view=azure-devops#what-is-a-pull-request-template

**Comment by MiKuehn** [comment]
You can read the template file and add the content into the description. You only have to convert the LF into \n.

**Comment by DaleMckeown** [comment]
Yeah for whatever reason when creating a PR using this task it doesn't select the template, even when other non pipeline PRs use the template fine.

like @MiKuehn  said, you can detect and read your template, and pass it into the PR `description` field. line endings are a problem, but can be worked around.
